src/datalakebundle/table/schema/SchemaChecker.py

```diff
@@ -30,7 +30,8 @@ def check(self, df: DataFrame, table_definition: TableDefinition):
             raise Exception(error_message)
 
     def check_metadata(self, table_definition: TableDefinition):
-        self.__metadata_checker.check(table_definition)
+        # self.__metadata_checker.check(table_definition)
+        pass
```

Comment:
Why pass?

Reply:
We haven't decided if we wanted to turn it on. So it's disabled